src/core/collection/Collection.tsx

```diff
@@ -159,14 +160,15 @@ function getEventCollectionCards(
     items,
     __typename: 'GeneralCollectionType',
   };
-  const cards = getCollectionCards(generalCollection).map((cardProps) => {
+  const cards = getCollectionCards(generalCollection).map((cardProps, i) => {
     const url = getRoutedInternalHref(cardProps.url, null);
     return (
       <Card
         key={cardProps.id}
         {...cardProps}
         url={url}
         direction="fixed-vertical"
+        customContent={<EventCardContent event={items[i]} />}
```

Comment:
trivial: event/item could be added to a new variable

Reply:
i ll try to fix, maybe typing as well